CorpusSearch/newdocs

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#note: this script needs to be run on the manx-search-data repo
+# not the repo it lives in
+# weirdest trick is the pattern matching to get dir from fdir
+# basically ## means delete the longest matching pattern from the var, # the shortest
+# for more on that see https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
```

Comment:
Move this to where the functionality is executed

Reply:
Split up the 'what' and the 'how'. Most users are reading to skim, so provide this information upfront

* Explain the intention of the code
* Give an example
* Explain how it works